docs/pip.md

```diff
@@ -484,6 +485,50 @@ hermeto-output/deps/pip
 └── ...
 ```
 
+## Working with Rust-based dependencies
+
+Hermeto provides a way to prepare hermetic build of a Python package which depends on
+on Rust-based packages. This process cannot be fully automated on Hermeto's side, thus it would
+require minor intervention from users.
+
+Building such project requires that all build dependencies are present. To achieve this
+[pybuild-deps](https://pypi.org/project/pybuild-deps) could be used. The following command
+will take care of generating the list:
+
+```
+echo <package_name==package_version> | pybuild-deps compile /dev/stdin --generate-hashes -o requirements-build.txt
+```
+
+where _package_name_ and _package_version_ are the name and the version of a package to be
+processed. Note, that this step requires internet connection.
+
+Once requirements-build.txt is populated fetch could be done as usual. Hermeto will fetch all
+runtime and build time dependencies for both Python and Rust parts.
+
+To build such Python package one would need to supply a proper `.cargo/config.toml`. It has
+to be provided manually because the build systems in use appear to be very sensitive to its
+placement which could change between fetch and build phases. `.cargo` directory must be
+created on the same level as `hermeto-output` directory and must contain a `config.toml`
+file of the following contents:
```

Comment:
Somehow I didn't connect the dots in my head when I saw this being done in the test Dockerfile, it was only I read this paragraph that made me realize that. Can't we achieve this in the inject-files command? We can make a note on rust extensions being used by the repo in the build config file and we know where to create it.

Reply:
The cargo [docs](https://doc.rust-lang.org/cargo/reference/config.html#hierarchical-structure) also indicate that we could make use of $CARGO_HOME to try to bypass this? I did not test it, though.

Reply:
> but breaking the isolation of the project folder would be new ground to us

How so? We inject to the prefetch dir for bundler as well as rpm.